### PR TITLE
Add AA4AM issue template to replace generic bug report

### LINKED ISSUE #489: [AA4AM] Align Issue Templates for ALM Accelerators with CoE Starter Kit
The generic issue templates we have now should align to the CoE Starter Kit Issue Templates to help guide users through issue creation and categorize issues better. ~~Additionally, need to clean up the Issue Template folder that got added to the ALMAcceleratorSampleSolution branch incorrectly~~.

### PR DESCRIPTION
**Pull request by jenschristianschroder**
Added AA4AM specific bug report template to help guide users through issue creation and categorize issues better.

fixes #489

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/alm-accelerator-for-advanced-makers-bug-report.md [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 ---
 
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+**Component (please tell us which part you are experiencing issues with):**
+
+- Version [e.g. 1.29]
+- Component [e.g. ALM Accelerator Canvas App]
+
```
Comment (mikefactorial) [inline .github/ISSUE_TEMPLATE/alm-accelerator-for-advanced-makers-bug-report.md:17]: Add ALM Accelerator Pipelines/Workflows to the e.g.

File: .github/ISSUE_TEMPLATE/alm-accelerator-for-advanced-makers-bug-report.md [hunk 2]
```diff
@@ -1,17 +1,23 @@
 ---
-name: Bug report
-about: Create a report to help us improve the CoE Starter Kit
-title: ''
-labels: ''
-assignees: ''
+name: "[ALM Accelerator] Bug report"
+about: Create a report to help us improve the ALM Accelerator for Advanced Makers
```
Comment (mikefactorial) [inline .github/ISSUE_TEMPLATE/alm-accelerator-for-advanced-makers-bug-report.md:3]: Change ALM Accelerator for Advanced Makers to ALM Accelerator to make it generic to both solutions

File: .github/ISSUE_TEMPLATE/alm-accelerator-for-advanced-makers-bug-report.md (hunk 1 above)
Comment (jenschristianschroder) [reply .github/ISSUE_TEMPLATE/alm-accelerator-for-advanced-makers-bug-report.md:17]: added this in latest commit

File: .github/ISSUE_TEMPLATE/alm-accelerator-for-advanced-makers-bug-report.md (hunk 2 above)
Comment (jenschristianschroder) [reply .github/ISSUE_TEMPLATE/alm-accelerator-for-advanced-makers-bug-report.md]: changed in latest commit

**Review by mikefactorial — APPROVED** [review]

**Pull request merged**